sortinghat/core/models.py

```diff
@@ -143,6 +145,18 @@ def __str__(self):
         return self.name
 
 
+class Group(MP_Node, EntityBase):
+    name = CharField(max_length=MAX_SIZE_CHAR_INDEX)
+    organization = ForeignKey(Organization, related_name='groups', on_delete=CASCADE, blank=True, null=True)
```

Comment:
It would be better to split this line in:

```
organization = ForeignKey(Organization, related_name='groups', on_delete=CASCADE,
                          blank=True, null=True)
```

Reply:
Got it, I forgot to format the file and break down lines with >80 chars, will do

Reply:
Done 👍